### DISCUSSION TITLE
Add the ATS25 max decoder receiver to the hamlib radio list.

### DISCUSSION THREAD

**Discussion by it9ybg** [Hamlib/Hamlib#1620] [General]
Hi, I hope this is the right section to ask if it is possible to add the ATS25 max decoder receiver  to the hamlib radio list. Thank you

**Comment by mdblack98** [comment]
I don't see any information on that radio having CAT control.
Is there some reason you think it does have it?

Mike W9MDB








On Monday, October 7, 2024 at 08:13:10 AM CDT, Giuseppe ***@***.***> wrote: 







Hi, I hope this is the right section to ask if it is possible to add the ATS25 max decoder receiver to the hamlib radio list. Thank you

**Comment by N0NB** [comment]
A quick search did not reveal any CAT capability of this receiver.  If support actually exists and documentation is publicly available, then open a new issue requesting support.
